### PR TITLE
Add CODEOWNERS for dbm

### PR DESCRIPTION
**Pull request by corona10**
Since there are no code owners for gdbm and ndbm module, I am adding codeowners who are familiar with codebase and possible to review related changes.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -42,6 +42,9 @@ Lib/test/test_type_*.py       @JelleZijlstra
 Lib/test/test_capi/test_misc.py  @markshannon @gvanrossum
 Tools/c-analyzer/             @ericsnowcurrently
 
+# dbms
+**/*dbm*                      @corona10 @erlend-aasland @serhiy-storchaka
```
Comment (corona10) [inline .github/CODEOWNERS:46]: Order is just alphabetical ordered.

Comment (corona10) [reply .github/CODEOWNERS:46]: Please let me know if you want to be the most preferred backend. ;)

**Review by erlend-aasland — APPROVED** [review]
Ok for me!

**Comment by corona10** [comment]
Wait @serhiy-storchaka too :)

**Review by serhiy-storchaka — APPROVED** [review]

**Pull request merged**